src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -294,6 +294,7 @@ def _encode_prompt(
         negative_prompt=None,
         prompt_embeds: Optional[torch.FloatTensor] = None,
         negative_prompt_embeds: Optional[torch.FloatTensor] = None,
+        lora_scale: Optional[float] = None,
```

Comment:
Wouldn't be more prudent to pass it from the `load_lora_weights()`. 

IIUC, `_encode_prompt()` is called internally in the pipeline and the end user doesn't have any access to it. So, I don't think this way we expose an explicit control to user to specify what scale they wanna use for the text encoder LoRA.